index.bs

```diff
@@ -443,10 +443,9 @@ To <dfn>handle an incoming message</dfn> given a [=WebSocket connection=] |conne
 To <dfn>emit an event</dfn> given |session|, and |body|:
 
   1. [=Assert=]: |body| has <a for=map>size</a> 2 and <a for=map>contains</a> "`method`" and "`params`".
```

Comment:
While editing, should the syntax for these links be modified into `[=map/size=]` and `[=map/exists=]`?

Reply:
Improving consistency sounds like a worthwhile improvement, but I'd prefer to do it in a dedicated patch, particularly since there are additional references that we'd need to update to address this completely:

    $ grep for=map index.bs
    1. Let |capabilities request| be |parameters|["`capabilities`"] if it <a for=map>exists</a>, else null.
    2. Let |required capabilities| be |capabilities request|["`alwaysMatch`"] if it <a for=map>exists</a>, else null.
        2. [=Assert=]: |matched| <a for=map>contains</a> "`id`", "`method`", and "`params`".
        2. If |parsed| is a [=map=] and |parsed|["`id`"] <a for=map>exists</a> and is an integer greater than or equal to zero, set |command id| to that integer.
        4. If |parsed| is a [=map=] and |parsed|["`method`"] <a for=map>exists</a> and is a string, but |parsed|["`method`"] is not in the [=set of all command names=], set |error code| to [=unknown command=].
      1. [=Assert=]: |body| has <a for=map>size</a> 2 and <a for=map>contains</a> "`method`" and "`params`".

Reply:
Here's a follow-up issue: https://github.com/w3c/aria-at-automation/issues/70